forms/wagtail_hooks.py

```diff
@@ -2,7 +2,7 @@
 from wagtail import hooks
 
 
-@hooks.register('insert_editor_css')
+@hooks.register('insert_global_admin_css')
```

Comment:
This CSS is definitely supposed to be used in the nested form fields in the form group of a form page. However, like you correctly pointed out, they are not getting applied because of the change in wagtail classname structure. Honestly, I don't feel like any of the CSS designs that we have mentioned here are needed (or even makes sense). The only one that kind of makes sense is for the multiple option field, we _can_ make it `width:100%`, but I don't know with the new design if it's really needed. CC: @harrislapiroff 

This is how it looks without width: 100%:
![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/9530293/c7b9862a-d11f-4b67-9727-54b69f2af2da)


And this is with width: 100%:
![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/9530293/acb4a03b-d4f7-4a0d-bb63-297be36d529f)

Reply:
My inclination here is to remove our CSS. The newer admin design seems like it doesn't need it anymore. We were just ahead of the curve, maybe :slightly_smiling_face:

Reply:
I agree. My vague recollection is that nested inlines used to be very poorly supported style-wise (maybe not supported) and we were probably accounting for that?